### ISSUE TITLE
Use ES6 import via vendor shim

### ISSUE THREAD

**Issue by jrowlingson** [joebartels/ember-cli-js-cookie#4]
Ideally, it would be nice to employ a [vendor-shim](https://ember-cli.com/user-guide/#standard-non-amd-asset) to avoid setting a global.

**Comment by joebartels** [comment]
Agree. It's sort of a fake benefit here since the addon is simply exporting the global. 
Perhaps this would work inside index.js 

```js
this.import('vendor/js.cookie.js', {
  using: [{ transformation: 'amd', as: 'js-cookie' }]
});
```

**Comment by jrowlingson** [comment]
I think I prefer this approach over using a vendor-shim. I took this for a quick test [here](https://github.com/jrowlingson/ember-cli-js-cookie/commit/1a233ccb341e7d114039e55de219bf1bcf960952) but I am observing the `Cookies` global is still being set on `window`.

I will take a closer look when I get a chance.

**Comment by joebartels** [comment]
ah I see... it may not be possible with the regular js-cookie build. 
Looking at the way the code is loaded:
js-cookie is looking for `define.amd`
https://github.com/js-cookie/js-cookie/blob/master/src/js.cookie.js#L10

and ember's amd loader has this mention...
https://github.com/ember-cli/loader.js/blob/master/lib/loader/loader.js#L239

**Comment by jrowlingson** [comment]
As of ember 2.15 `app.import` [supports](https://www.emberjs.com/blog/2017/09/01/ember-2-15-released.html#toc_app-import-files-within-node_modules) importing files from `node_modules`
